extras/rya.console/src/main/java/mvm/rya/shell/RyaConnectionCommands.java

```diff
@@ -154,24 +158,27 @@ public String connectToAccumulo(
 
     @CliCommand(value = CONNECT_MONGO_CMD, help = "Connect the shell to an instance of Mongo.")
     public String connectToMongo(
-            @CliOption(key = {"username"}, mandatory = true, help = "The username that will be used to connect to Accummulo.")
+            @CliOption(key = {"username"}, mandatory = true, help = "The username that will be used to connect to Mongo.")
             final String username,
-            @CliOption(key = {"instanceName"}, mandatory = true, help = "The name of the Accumulo instance that will be connected to.")
+            @CliOption(key = {"hostname"}, mandatory = true, help = "The hostname where Mongo resides.")
+            final String hostname,
+            @CliOption(key = {"instanceName"}, mandatory = true, help = "The name of the Mongo instance that will be connected to.")
```

Comment:
What is a Mongo instance name? We only have this for Accumulo because the Accumulo instance itself has a name within zookeeper.